server/lib/job-queue/handlers/video-upload-tmp-folder-cleaner.ts

```diff
@@ -0,0 +1,53 @@
+import { readdir, unlink, stat } from 'fs-extra'
+import { Stats } from 'node:fs'
+import * as path from 'path'
+import { logger } from '@server/helpers/logger'
+import { getTmpPath } from '@server/helpers/utils'
+import { SCHEDULER_INTERVALS_MS } from '@server/initializers/constants'
+
+async function processVideoUploadTmpFolderCleaner () {
+  const tmpPath = await getTmpPath()
+  let files: string[]
+
+  logger.debug('processVideoUploadTmpFolderCleaner: Reading files from path %s', tmpPath)
+
+  try {
+    files = await readdir(tmpPath)
```

Comment:
If you created the folder at peertube startup, this function should not fail so you don't need to try catch this case